frontend/packages/dev-console/src/components/edit-deployment/deployment-strategy/utils/deployment-strategy-utils.ts

```diff
@@ -1,5 +1,63 @@
 import { TFunction } from 'i18next';
-import { LifecycleAction } from './types';
+import { Resources } from '../../../import/import-types';
+import { DeploymentStrategyType, LifecycleAction } from './types';
+
+export const getDeploymentStrategyItems = (resourceType: Resources, t: TFunction) => {
+  switch (resourceType) {
+    case Resources.Kubernetes:
+      return {
+        [DeploymentStrategyType.recreateParams]: t('devconsole~Recreate'),
+        [DeploymentStrategyType.rollingUpdate]: t('devconsole~Rolling Update'),
```

Comment:
```suggestion
        [DeploymentStrategyType.rollingUpdate]: t('devconsole~Rolling update'),
```

Reply:
Hmm it was "Rolling Update" before. It's fine for me to change this, but the detail page shows this:


![image](https://user-images.githubusercontent.com/139310/127360088-9718b624-e6aa-4fc4-aa57-171e3ae6b3be.png)


And this is the "Edit Deployment" page at the moment:


![image](https://user-images.githubusercontent.com/139310/127360181-c873b6bb-d958-44b7-ba3a-49e17e7b036b.png)


@divyanshiGupta and @rebeccaalpert Wdyt?

Reply:
What do you think @spadgett?